webapp/web/configuration.css

```diff
@@ -206,8 +206,9 @@
   overflow-y: scroll;
   border: 1px solid #ddd;
   border-radius: .5rem;
-  padding: 2px;
-  min-height: 70px;
+  padding: 3px;
+  width: 95%;
+  height: 75px;
```

Comment:
I see this changed from `min-height` to `height`, but I don't think it's what we want? the previous behaviour in terms of height is what we want - start with a minimum, and expand based on contents, so can you please revert this? thanks!

Reply:
Reverted it back

Reply:
thanks! unfortunately it doesn't seem to behave as it did - if I write something, the suggested replies box I'm writing in is expanding correctly, but the translation is somehow shorter height, and expands slower. I think they should all be the same height if possible?

Reply:
I've been able to make them expand the same time